promising/promising_context.py

```
@@ -560,87 +602,84 @@ def schedule_await_children() -> None:
+        return result
+
+    def get_thread_pool_executor(self) -> concurrent.futures.ThreadPoolExecutor | None:
+        """
+        Return the thread pool executor for ``loop.run_in_executor``.
+        """
+        return self._thread_pool
 
     def __enter__(self) -> "PromisingContext":
         if self._previous_token is not None:
-            raise ContextAlreadyActiveError("This PromisingContext is already active")
+            raise ContextAlreadyActiveError(f"{self!r} is already active")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)